.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: ['1.18', '1.19']
+        go: ['1.21']
```

Comment:
I think it's not the best time to drop support for golang<1.21 for now

Reply:
go 1.21 and 1.22 is the only supported Stable versions now. https://go.dev/dl/

is this OK now?